### PR TITLE
WIP: Fix zoom speed

### PR DESCRIPTION
**Pull request by peetCreative**
Approaches #903 and #908

### CODE REVIEW

File: src/control/zoom/ZoomControl.cpp [hunk 1]
```diff
@@ -35,6 +40,42 @@ ZoomControl::~ZoomControl()
...
 }
 
+void ZoomControl::zoomOneStep(bool zoomIn, double x, double y)
+{
+	XOJ_CHECK_TYPE(ZoomControl);
+
+	startZoomSequence(x, y);
+
+	if(zoomIn)
+		this->zoom += this->zoomStep;
```
Comment (andreasb242) [inline src/control/zoom/ZoomControl.cpp]: Missing Brackets

**Comment by peetCreative** [comment]
Please do not merge it's not finished

**Comment by cbm755** [comment]
Suggestion: common practice to put 'WIP' in PR title in this case

**Comment by andreasb242** [comment]
Agree to the WIP (Work in progress), I also saw this on other projects.

**Comment by andreasb242** [comment]
@peetCreative Thank you!

**Pull request merged**